src/databricks/sqlalchemy/test/_unsupported.py

```diff
@@ -55,6 +55,7 @@ class SkipReason(Enum):
     TIMEZONE_OPT = "timezone-optional TIMESTAMP fields"
     TRANSACTIONS = "transactions"
     UNIQUE = "UNIQUE constraints"
+    DROP_TBL = "drop table comment"
```

Comment:
Since `''` is identical to `None` as far as sqlalchemy is concerned, I think we needn't list this as unsupported. We can just document the behaviour. Thoughts?